### PR TITLE
Encointer-Testnet to run with encointer community-sidechain

### PR DESCRIPTION
**Pull request by echevrier**
- Add Integritee teerex and sidechain pallets
- Add skip-ias-check feature

### CODE REVIEW

File: runtime/Cargo.toml [hunk 1]
```diff
@@ -26,6 +26,8 @@ pallet-encointer-ceremonies-rpc-runtime-api = { default-features = false, git =
 pallet-encointer-communities = { default-features = false, git = "https://github.com/encointer/pallets", branch = "master" }
 pallet-encointer-communities-rpc-runtime-api = { default-features = false, git = "https://github.com/encointer/pallets", branch = "master" }
 pallet-encointer-scheduler = { default-features = false, git = "https://github.com/encointer/pallets", branch = "master" }
+pallet-sidechain = { default-features = false, git = "https://github.com/integritee-network/pallets.git", branch = "develop" }
+pallet-teerex = { default-features = false, git = "https://github.com/integritee-network/pallets.git", branch = "develop" }
```
Comment (clangenb) [inline runtime/Cargo.toml:31]: Is this on purpose the develop branch?

**Review by clangenb — COMMENTED** [review]
I have one question, but it looks good otherwise

File: runtime/Cargo.toml (hunk 1 above)
Comment (echevrier) [reply runtime/Cargo.toml:31]: It was, when I did it ;)
As the pallet mastrer branch is fixed, I will change it

Comment (echevrier) [reply runtime/Cargo.toml:31]: Actually, I think polkadot-v.0.9.28 is only working with the develop branch. So we should change it when we do a polkadot upgrade

**Review by clangenb — APPROVED** [review]
Looks good to me now!

**Pull request merged**